pkg/providers/gcp/gcp.go

```diff
@@ -72,7 +78,7 @@ func New(
 	// it would try authenticate against google regardless of overwriting the
 	// client
 	if g.client == nil {
-		c, err := monitoring.NewQueryClient(ctx)
+		c, err := monitoring.NewQueryClient(context.TODO(), clientOptions...)
```

Comment:
Im not sure why this is changed to `context.TODO()`?

Reply:
Because in the future the timeout should be defined for each account configuration